File: dotnet/logbrew-dotnet/examples/ReadmeExample.cs
using System;
using LogBrew;

public static class Program
{
    public static void Main()
    {
        var client = LogBrewClient.Create("LOGBREW_API_KEY", "logbrew-dotnet", "0.1.0");
        EnqueueAll(client);

        Console.WriteLine(client.PreviewJson());
        var response = client.Shutdown(RecordingTransport.AlwaysAccept());
        Console.Error.WriteLine("{\"ok\":true,\"status\":" + response.StatusCode + ",\"attempts\":" + response.Attempts + ",\"events\":6}");
    }

    private static void EnqueueAll(LogBrewClient client)
    {
        client.Release("evt_release_001", "2026-06-02T10:00:00Z", ReleaseAttributes.Create("1.2.3").WithCommit("abc123def456").WithNotes("Public release marker"));
        client.Environment("evt_environment_001", "2026-06-02T10:00:01Z", EnvironmentAttributes.Create("production").WithRegion("global"));
        client.Issue("evt_issue_001", "2026-06-02T10:00:02Z", IssueAttributes.Create("Checkout timeout", "error").WithMessage("Request timed out after retry budget"));
        client.Log("evt_log_001", "2026-06-02T10:00:03Z", LogAttributes.Create("worker started", "info").WithLogger("job-runner"));
        client.Span("evt_span_001", "2026-06-02T10:00:04Z", SpanAttributes.Create("GET /health", "trace_001", "span_001", "ok").WithDurationMs(12.5));
        client.Action("evt_action_001", "2026-06-02T10:00:05Z", ActionAttributes.Create("deploy", "success"));
    }
}

File: kotlin/logbrew-kotlin/examples/readme_example/ReadmeExample.kt
import co.logbrew.sdk.ActionAttributes
import co.logbrew.sdk.EnvironmentAttributes
import co.logbrew.sdk.IssueAttributes
import co.logbrew.sdk.LogAttributes
import co.logbrew.sdk.LogBrewClient
import co.logbrew.sdk.RecordingTransport
import co.logbrew.sdk.ReleaseAttributes
import co.logbrew.sdk.SpanAttributes

fun main() {
    val client =
        LogBrewClient.create(
            apiKey = "LOGBREW_API_KEY",
            sdkName = "logbrew-kotlin",
            sdkVersion = "0.1.0",
        )
    enqueueCanonicalEvents(client)
    println(client.previewJson())
    val response = client.flush(RecordingTransport.alwaysAccept())
    System.err.println("""{"ok":true,"status":${response.statusCode},"attempts":${response.attempts},"events":6}""")
}

fun enqueueCanonicalEvents(client: LogBrewClient) {
    client.release(
        "evt_release_001",
        "2026-06-02T10:00:00Z",
        ReleaseAttributes.create("1.2.3").withCommit("abc123def456").withNotes("Public release marker"),
    )
    client.environment(
        "evt_environment_001",
        "2026-06-02T10:00:01Z",
        EnvironmentAttributes.create("production").withRegion("global"),
    )
    client.issue(
        "evt_issue_001",
        "2026-06-02T10:00:02Z",
        IssueAttributes.create("Checkout timeout", "error").withMessage("Request timed out after retry budget"),
    )
    client.log(
        "evt_log_001",
        "2026-06-02T10:00:03Z",
        LogAttributes.create("worker started", "info").withLogger("job-runner"),
    )
    client.span(
        "evt_span_001",
        "2026-06-02T10:00:04Z",
        SpanAttributes.create("GET /health", "trace_001", "span_001", "ok").withDurationMs(12.5),
    )
    client.action(
        "evt_action_001",
        "2026-06-02T10:00:05Z",
        ActionAttributes.create("deploy", "success"),
    )
}
